src/execution/operator/aggregate/physical_window.cpp

```diff
@@ -230,8 +230,8 @@ class WindowGlobalSinkState : public GlobalSinkState {
 
 	//! Parent operator
 	const PhysicalWindow &op;
-	//! Execution context
-	ClientContext &context;
+	//! Client context
+	ClientContext &client;
```

Comment:
nit: should we keep calling this `context` to stay consistent with existing code? Same in other places in this PR. If its not too much work given the amount of changes to `client` 😅